### PR TITLE
Update Puppet documents

### PR DESCRIPTION
**Pull request by Zenidd**
Hello,

This PR updates the puppet documents for the upcoming v4.0 that is about to be released soon. 

Greetings,

JP

### CODE REVIEW

File: source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-agent-class.rst [hunk 1]
```diff
@@ -5,146 +5,717 @@
...
+  `Default yes`
+
+$ossec_rootcheck_check_if
+  Enable rootcheck checkif option.
+
+  `Default yes`
+
 $ossec_rootcheck_frequency
-  Setup rootcheck frequency.
+  Frequency that the rootcheck is going to be executed (in seconds).
```
Comment (manuasir) [inline source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-agent-class.rst:265]:
```suggestion
  How often the rootcheck scan will run (in seconds).
```

File: source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-agent-class.rst [hunk 2]
```diff
@@ -5,146 +5,717 @@
...
+
+  `Default 12h`
+
+$sca_rhel_skip_nfs
+  Enable or disable the scanning of network mounted filesystems (Works on Linux and FreeBSD). Currently, `skip_nfs` will exclude checking files on CIFS or NFS mounts.
+
+  `Default yes`
+
+$sca_rhel_policies
+  In this section it can be included a list of policy files to run assessments.
```
Comment (manuasir) [inline source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-agent-class.rst:370]:
```suggestion
  A list of policies to run assessments can be included in this section.
```

**Review by manuasir — CHANGES_REQUESTED** [review]

File: source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-agent-class.rst (hunk 1 above)
Comment (Zenidd) [reply source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-agent-class.rst]: Done on 2faa2ea, thanks !

File: source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-agent-class.rst (hunk 2 above)
Comment (Zenidd) [reply source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-agent-class.rst]: Done on 2faa2ea, thanks !

**Review by manuasir — APPROVED** [review]

**Review by xr09 — APPROVED** [review]
LGTM

**Review by davidfmiranda — APPROVED** [review]
LGTM

**Pull request merged**